api/src/org/apache/cloudstack/api/command/admin/vlan/UpdateVlanIpRangeCmd.java

```diff
@@ -0,0 +1,137 @@
+package org.apache.cloudstack.api.command.admin.vlan;
```

Comment:
license header is missing, this won't pass the RAT

Reply:
added